.github/workflows/storybook.yml

```diff
@@ -5,7 +5,35 @@ on:
     branches:
       - develop
       - main
+    # Only run if CSS/JS/MD-related files changed.
+    paths:
+      - '.github/workflows/storybook.yml'
+      - 'assets/**'
+      - 'packages/**'
+      - 'tests/backstop/**'
+      - '.storybook/**'
+      - '!.storybook/storybook-data.js'
+      - '**.scss'
+      - '.eslint*'
+      - '.nvmrc'
+      - '.prettier*'
+      - '**/package.json'
+      - 'package-lock.json'
   pull_request:
+    # Only run if CSS/JS/MD-related files changed.
+    paths:
+      - '.github/workflows/storybook.yml'
+      - 'assets/**'
+      - 'packages/**'
+      - 'tests/backstop/**'
```

Comment:
Storybook changes are relevant to VRT, but backstop changes aren't relevant for Storybook 😄 
```suggestion
```

Reply:
Hah, right, my brain got turned around 😆